examples/datasets/plot_limo_data.py

```diff
@@ -111,68 +169,250 @@
 # We can also compute the difference wave contrasting Face A and Face B.
 # Although, looking at the evoked responses above, we shouldn't expect great
 # differences among these face-stimuli.
-#
-# Compute difference wave (Face A minus Face B)
-difference_wave = mne.combine_evoked([limo_epochs['Face/A'].average(),
-                                     -limo_epochs['Face/B'].average()],
-                                     weights='equal')
 
-# Plot difference between Face A and Face B
+# Face A minus Face B
+difference_wave = combine_evoked([limo_epochs['Face/A'].average(),
+                                  -limo_epochs['Face/B'].average()],
+                                 weights='equal')
+
+# plot difference wave
 difference_wave.plot_joint(times=[.15], title='Difference Face A - Face B')
 
 ###############################################################################
-# As expected, no see clear differential patterns appears when contrasting
-# Face A and Face B. However, we could narrow our search to
-# since this is a "visual paradigm" it might be best to electrodes located over
-# the occipital lobe. After all this is "visual paradigm". Thus, differences
-# between stimuli (if any) might easier to spot over "more visual areas".
-#
+# As expected, no see clear differential pattern appears when contrasting
+# Face A and Face B. However, we could narrow our search a little bit more.
+# Since this is a "visual paradigm" it might be best to look at electrodes
+# located over the occipital lobe, as differences between stimuli (if any)
+# might easier to spot over visual areas.
+
 # Create a dictionary containing the evoked responses
 conditions = ["Face/A", "Face/B"]
 evoked_dict = dict()
 for condition in conditions:
     evoked_dict[condition] = limo_epochs[condition].average()
```

Comment:
I would always do `evokeds = {condition: limo_epochs[condition].average() for condition in conditions}`